src/metabase/middleware.clj

```diff
@@ -168,6 +173,11 @@
    "Expires"        "Tue, 03 Jul 2001 06:00:00 GMT"
    "Last-Modified"  (du/format-date :rfc822)})
 
+ (defn- cache-far-future-headers
+   "Headers that tell browsers to cache a static resource for a long time."
+   []
+   {"Cache-Control" "public, max-age=31536000"})
```

Comment:
This can just be a `def`

Reply:
Again I was just copying what `cache-prevention-headers` did, which also doesn't seem like it needs to be a function, but is for some reason.

Reply:
`cache-prevention-headers` needs to be a function because it includes the time of the request as the `Last-Modified` value, which is different for every request. `cache-far-future-headers` is basically just a constant and doesn't change on each request.